### PR TITLE
updated GCE instructions 2018

### PR DESCRIPTION
**Pull request by jhpenger**

### CODE REVIEW

File: worker.yml [hunk 1]
```diff
@@ -6,21 +6,33 @@ metadata:
   labels:
     app: ray-worker
 spec:
-  replicas: 3
+  replicas: 67
```
Comment (cioc) [inline worker.yml]: maybe let's make this 3 for the generic code?

File: test_cluster.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
+from __future__ import absolute_import
```
Comment (cioc) [inline test_cluster.py:1]: love the test!

**Review by cioc — COMMENTED** [review]
Great work!  Maybe change the replica value back to 3 then we can pull this in?  Thanks!

**Review by cioc — APPROVED** [review]

**Comment by jhpenger** [comment]
Robert suggested using pip for installing ray. see [here](https://github.com/robertnishihara/ray-kubernetes/pull/1)
I am a bit confused my self regarding kubernetes pods. It seems like each pod is requesting `100 miliCPU` resource; but when I access pods directly, they each show resource of `4 vCPU & 2 cores` (assuming node hosting the pod was setup with `4 vCPU`). The `test_cluster` I modified from `exercise04` confirms that the compute power is `2 cores` each. I am not familiar with CPU threading/scheduling, any idea how `100m CPU` turned into `4 CPU 2 cores"?

Also, have you any experience with setting up GPU Kubernetes clusters?

**Pull request merged**